### PR TITLE
Add animals table

### PR DESCRIPTION
**Pull request by teckim**
### In this pull request:

- Created `vet_clinic` database.
- Created `animals` table and schema.
- Inserted values into the `animals` table.
- Wrote queries against the `animals` table.

**SCREENSHOTS:**

> Screen Shot 2022-08-08 at 4 05 03 PM
> 
> Screen Shot 2022-08-08 at 4 05 33 PM
> 
> Screen Shot 2022-08-08 at 4 06 01 PM
> 
> Screen Shot 2022-08-08 at 4 06 20 PM
> 
> Screen Shot 2022-08-08 at 4 06 40 PM
> 
> Screen Shot 2022-08-08 at 4 07 04 PM
> 
> Screen Shot 2022-08-08 at 4 07 22 PM
> 
> Screen Shot 2022-08-08 at 4 07 44 PM
> 
>

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INTEGER GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg REAL,
```
Comment (omar-muhamad) [inline schema.sql:9]: - [ ] [Optional] It would be better if you used `boolean` data type instead of `real` data type.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = TRUE;
+SELECT * FROM animals WHERE name <> 'Gabumon';
+SELECT * FROM animals WHERE round(weight_kg::numeric, 2) >= 10.23 AND round(weight_kg::numeric, 2) <= 17.3;
```
Comment (omar-muhamad) [inline queries.sql:10]: - [ ] [Optional] You don't have to round the numbers before comparing them, you can simply do it like that:
```sql
SELECT * FROM animals WHERE weight_kg >= '10.4' AND weight_kg <= '17.3';
```

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @teckim,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### ⚠️Please consider the Optional suggestions below⚠️



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (teckim) [reply schema.sql:9]: @Omar-Muhamad `boolean` instead of `real`!!!

File: queries.sql (hunk 2 above)
Comment (teckim) [reply queries.sql:10]: @Omar-Muhamad that way 10.4 and 17.3 will not be included.

**Pull request merged**